daft/expressions/expressions.py

```diff
@@ -750,6 +760,24 @@ def hash(self, seed: Any | None = None) -> Expression:
             expr = self._expr.hash(seed._expr)
         return Expression._from_pyexpr(expr)
 
+    def shift_left(self, bits: int | Expression) -> Expression:
+        """Shifts each element of the list to the left by a specified number of bits
+
+        Returns:
+            Expression: an expression with the type of the list values left shifted by bits
+        """
+        bits_expr = Expression._to_expression(bits)
+        return Expression._from_pyexpr(self._expr.shift_left(bits_expr._expr))
+
+    def shift_right(self, bits: int | Expression) -> Expression:
+        """Shifts each element of the list to the right by a specified number of bits
```

Comment:
Can you add a comment here to specify that: 
For unsigned integers, this expression should perform a logical right shift.
For signed integers, this expression should perform an arithmetic right shift.